recipes/bootstrap_contrast/meta.yaml

```diff
@@ -0,0 +1,63 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
+
+# Jinja variables help maintain the recipe as you'll update the version only here.
+{% set name = "bootstrap_contrast" %}
+{% set version = "0.320.1" %}
+{% set sha256 = "923b3f470085dbdd465ed6cda82440b341362c552a4189f8c44496ed422a3276" %}
+# sha256 is the prefered checksum -- you can get it for a file with:
+#  `openssl sha256 <file name>`.
+# You may need the openssl package, available on conda-forge
+#  `conda install openssl -c conda-forge``
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+  
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+  run:
+    - python
```

Comment:
You should add all of the packages that are run time requirements of `bootstrap_contrast` here. Probably everything listed:

https://github.com/josesho/bootstrap_contrast/blob/master/setup.py#L16-L42

including the proper version restrictions.